bookdown/07-special-density.rmd

```diff
@@ -0,0 +1,89 @@
+## Density Estimation {#density}
+
+Density estimation is the learning task to find the unknown distribution from which an i.i.d. data set is generated. We interpret this broadly, with this distribution not necessarily being continuous (so may possess a mass not density). The conditional case, where a distribution is predicted conditional on covariates, is known as ‘probabilistic supervised regression’, and will be implemented in `r mlr_pkg("mlr3proba")` in the near-future. Unconditional density estimation is viewed as an unsupervised task.
```

Comment:
Nitpicking here, but we agreed on trying to have one sentence per line. This greatly simplifies reading of diffs.

Reply:
Also makes it possible to comment on sentences instead of paragraphs. ;)